site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -258,6 +258,7 @@ A device MUST use the following source address in its *attach* frame to open a l
 
 * `command` (authenticated device)
 * `command/${tenant}/${device-id}` (unauthenticated device)
+* `command` (authenticated gateway)
```

Comment:
what about an unauthenticated gateway? Why not simply amend the existing lines with *device/gateway*?

Reply:
As far as I see it, the concept of an unauthenticated gateway is generally not supported in Hono, or is it?
I've amended the first line by adding "/gateway".

Reply:
hmm, I guess you are right. Without authentication we are not able to determine the gateway's device ID. Have we explicitly stated that somewhere in the docs?

Reply:
As far as I see, it isn't *explicitly* mentioned in the docs.
We could add a corresponding note in the protocol adapter user guides in the `Device authentication` chapters. I can have a look at that.